app/src/main/java/com/dankideacentral/dic/util/ImageProcessor.java

```diff
@@ -0,0 +1,44 @@
+package com.dankideacentral.dic.util;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by srowhani on 11/13/16.
+ */
+
+public class ImageProcessor {
+    /**
+     * Produces a mutable Bitmap fro a given request url.
+     * @param imageUrl
+     * @return
+     */
+    public static Bitmap bitmapFromUrl (String imageUrl) {
```

Comment:
I like this refactor - it was exactly the change I was gonna make if I did the feature ;)